migrations/20250409112337_subscriptions.sql

```diff
@@ -0,0 +1,11 @@
+-- create subscription table if it does not exits
+--  with timestamp to keep track of when a user logged in 
+CREATE TABLE
+    IF NOT EXISTS subscriptions (
+        id UUID PRIMARY KEY DEFAULT gen_random_uuid (),
+        name TEXT NOT NULL,
+        email TEXT NOT NULL UNIQUE,
+        subscribed_at TIMESTAMP
+        WITH
+            TIME ZONE DEFAULT now ()
```

Comment:
Postgres offers a convenient type for this: `timestamptz` which is a timezone aware date and time type.
```suggestion
        subscribed_at timestamptz NOT NULL
```